docs/pages/api/create-aptos-token.ts

```diff
@@ -0,0 +1,96 @@
+import { AptosAccount, AptosClient, HexString, TokenClient } from 'aptos';
+
+const NODE_URL = 'https://fullnode.devnet.aptoslabs.com';
+
+import { NextApiRequest, NextApiResponse } from 'next';
+
+import { ApiError } from '../../lib/error';
+
+export type CreateAptosTokenBody = {
+  receiver: string;
+  metadataUri: string;
+};
+
+export type CreateAptosTokenResponse = {
+  creator: string;
+  collectionName: string;
+  tokenName: string;
+  tokenPropertyVersion: number;
+};
+
+const handler = async (
+  req: NextApiRequest,
+  res: NextApiResponse<CreateAptosTokenResponse | ApiError>,
+) => {
+  try {
+    const method = req.method;
+
+    if (method === 'POST') {
+      const { receiver, metadataUri }: CreateAptosTokenBody = req.body;
+
+      if (!receiver || !metadataUri) {
+        return res.status(400).json({ message: 'Missing data in body.' });
+      }
+
+      const client = new AptosClient(NODE_URL);
+      const tokenClient = new TokenClient(client);
+
+      if (!process.env.APTOS_PRIVATE_KEY) {
+        return res.status(500).json({ message: 'Aptos config missing.' });
+      }
+
+      const issuer = new AptosAccount(
+        new HexString(process.env.APTOS_PRIVATE_KEY).toUint8Array(),
+      );
+
+      const collectionName = 'Livepeer';
+      const tokenPropertyVersion = 0;
+
+      const collectionData = await tokenClient.getCollectionData(
+        issuer.address(),
+        collectionName,
+      );
+
+      const tokenName = `Video NFT ${Number(collectionData?.supply ?? 0) + 1}`;
+
+      const txnHash2 = await tokenClient.createToken(
+        issuer,
+        collectionName,
+        tokenName,
+        tokenName,
```

Comment:
Is there a reason for `tokenName` twice?

Reply:
I'll clean this up! Should be a description